.env

```diff
@@ -1 +1,2 @@
+REACT_APP_VERSION=$npm_package_version
```

Comment:
https://create-react-app.dev/docs/adding-custom-environment-variables/#expanding-environment-variables-in-env